### DISCUSSION TITLE
Glossary in quarto?

### DISCUSSION THREAD

**Discussion by sn248** [quarto-dev/quarto-cli#1763] [Q&A]
Hi

As I understand, `rmarkdown` did not natively support creation of glossary. Is this something that is supported in quarto or is planned to be supported in future?

**Comment by jjallaire** [comment]
Yes, we will definitely implement support for glossaries (likely sometime over the next 9-12 months)

**Reply by sn248** [reply]
Thank you so much for the all good work Posit does!

**Reply by salim-b** [reply]
Feature request is tracked in https://github.com/quarto-dev/quarto-cli/issues/1697.

**Comment by lcnbr** [comment]
One of the things I would love to see is symbol glossaries for mathematical symbols. One could hover over a symbol in an equation, get an immediate tooltip explaining its meaning, and maybe even click it to open a link to a glossary entry describing it.

**Comment by Minh-AnhHuynh** [comment]
In the meantime, I have tried using acronymsdown package. But it seems Quarto doesn't recognize the command `\acr{<add acronym here>}`. Strangely the pandoc filter command seems to not make Quarto crash so I assume it is fine. Is there possibly a workaround ?

**Reply by lcnbr** [reply]
I was able to make it work not sure what I've done differently though...

**Reply by Minh-AnhHuynh** [reply]
Update on my previous replies : I used the acronym package and it actually works decently. Only issue I have is that greek letters in unicode absolutely do not work and I have to make due with using a weird specific font for greek letters such as `\mupalpha` from the upgreek package.

**Comment by mcanouil** [comment]
Closing in favour of https://github.com/quarto-dev/quarto-cli/issues/1697
